integration-tests/helpers/admin-seeder.js

```diff
@@ -1,5 +1,5 @@
 const Scrypt = require("scrypt-kdf")
-const { User } = require("@medusajs/medusa")
+const { User } = require("@medusajs/medusa/dist/models/user")
```

Comment:
Why was this change needed?

Reply:
Because importing from the entry file is causing issues since all import will get evaluated, meaning all models to and therefore some decorators relying on the feature flags gets applied before they are loaded